numba/tests/test_smart_array.py

```diff
@@ -57,8 +59,14 @@ def test_ufunc(self):
         cfunc = jit(nopython=True)(npyufunc_usecase)
         aa = cfunc(a)
         self.assertIsInstance(aa, SmartArray)
-        self.assertPreciseEqual(aa.host(), np.cos(np.sin(a.host())))
+        self.assertPreciseEqual(aa.get('host'), np.cos(np.sin(a.get('host'))))
 
+    def test_astype(self):
+        a = SmartArray(np.int32([42, 8, -5]))
+        #cfunc = jit(nopython=True)(astype_usecase)
```

Comment:
oups, forgot to reenable (after fixing the pythonapi bug). Thanks for catching that.